cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/includes/bottom.jsp

```diff
@@ -23,30 +23,45 @@
 
 </div>  <!-- END #content -->
 
+<spring:eval var="serverName" expression="@casProperties.getProperty('server.name')"/>
+<spring:eval var="osfLoginUrl" expression="@casProperties.getProperty('cas.osf.login.url')"/>
+<spring:eval var="casLogoutUrl" expression="@casProperties.getProperty('cas.logout.url')"/>
+<spring:eval var="createAccountUrl" expression="@casProperties.getProperty('osf.createAccount.url')"/>
+<spring:eval var="osfUrl" expression="@casProperties.getProperty('osf.url')"/>
```

Comment:
From line 27 to 30, move all spring evaluation to the top so they can be used by all conditions.